client/store/activeOrder.js

```diff
@@ -22,12 +24,51 @@ export const fetchUserOrder = () => {
     dispatch(getUserOrder(order))
   }
 }
+export const addToOrder = (product, count) => async dispatch => {
+  const {data} = await axios.get('/auth/me')
+  count = Number(count)
+  if (data.id) {
+    product.count = count
+    let orderRes = await axios.get(`/api/orders/user/${data.id}`)
+    let order = orderRes.data
+    await axios.put(`/api/orders/${order.id}`, product)
+    orderRes = await axios.get(`/api/orders/user/${data.id}`)
+    dispatch(addToCart(orderRes.data))
+  } else {
+    let cart = JSON.parse(localStorage.getItem('cart'))
+    if (cart) {
+      let isInCart = false
+      for (let i = 0; i < cart.products.length; i++) {
+        let currProd = cart.products[i]
+        if (currProd.id === product.id) {
+          currProd.OrderProducts.count += count
+          cart.totalPrice += count * product.price
+          isInCart = true
+        }
+      }
+      if (!isInCart) {
+        product.OrderProducts = {count: count}
+        cart.totalPrice += count * product.price
+        cart.products.push(product)
+      }
+      localStorage.setItem('cart', JSON.stringify(cart))
+    } else {
+      cart = {products: [], totalPrice: 0}
+      product.OrderProducts = {count: count}
+      cart.products.push(product)
+      cart.totalPrice = product.price * count
```

Comment:
would this be += here?

Reply:
This could really be either just because this is really where you're initializing the local cart!